src/__test__/pages/experiments/[experimentId]/plots-and-tables/heatmap/index.test.jsx

```diff
@@ -124,4 +111,17 @@ describe('Heatmap plot', () => {
 
     expect(screen.getByText(/Add some genes to this heatmap to get started/i)).toBeInTheDocument();
   });
+
+  it('Changing clusters updates the plot data', async () => {
+    await renderHeatmapPage(storeState);
+
+    // Open the Select Data panel
+    userEvent.click(screen.getByText(/Select data/i));
+
+    // Change from Louvain to Custom cell sets
+    userEvent.click(screen.getByText(/Louvain/i));
+    userEvent.click(screen.getByText(/Custom cell sets/i), null, { skipPointerEventsCheck: true });
```

Comment:
why do we need this `skipPointerEventsCheck` here?

Reply:
Options in the `antd` list is generated with `pointer-events:none` to prevent unintentional clicking when the dropdown is being rendered. However, in tests, this property is not removed in tests due to some JSDOM compatibility issues. There are some open issues on this, here is one:
https://github.com/ant-design/ant-design/issues/31105

One way to overcome this is to use the `fireEvent` function. However, the `userEvent` library also provides a way to overcome that:
https://testing-library.com/docs/ecosystem-user-event/#clickelement-eventinit-options